client/js/app/modules/api_query_url.js

```diff
@@ -0,0 +1,71 @@
+var _ = require('lodash');
+var Qs = require('qs');
+var ExplorerUtils = require('../utils/ExplorerUtils');
+var TimeframeUtils = require('../utils/TimeframeUtils');
+
+function ApiQueryUrl(client, explorer) {
```

Comment:
I think I had grand schemes of breaking out the whole function into smaller private functions, which I guess is possible without creating a new object. You're right, I can just export one function and avoid creating a new object.